h2/src/main/org/h2/command/Parser.java

```diff
@@ -1568,6 +1568,9 @@ private Prepared parseShow() {
         } else if (readIf("SERVER_ENCODING")) {
             // for PostgreSQL compatibility
             buff.append("'UTF8' AS SERVER_ENCODING FROM DUAL");
+        } else if (readIf("SSL")) {
+            // for PostgreSQL compatibility
+            buff.append("'off' AS SSL FROM DUAL");
```

Comment:
Please, merge changes from the `master` branch and use `"'off' SSL"` here.

Reply:
merged